lldb/source/Plugins/Disassembler/LLVMC/DisassemblerLLVMC.cpp

```diff
@@ -1446,6 +1446,33 @@ bool DisassemblerLLVMC::MCDisasmInstance::IsAuthenticated(
   return InstrDesc.isAuthenticated() || IsBrkC47x;
 }
 
+void DisassemblerLLVMC::UpdateFeatureString(llvm::StringRef additional_features,
+                                            std::string &features) {
+  // Allow users to override default additional features.
+  for (llvm::StringRef flag : llvm::split(additional_features, ",")) {
+    flag = flag.trim();
+    if (flag.empty())
+      continue;
+    // Check if disable (-c) is already present before adding default enable
+    // (+c), since enable (+c) takes precedence whether disable (-c) appears
+    // before or after in the feature string.
+    if (flag.starts_with('+')) {
+      std::string disable_flag = "-" + flag.substr(1).str();
+      if (features.find(disable_flag) == std::string::npos) {
+        if (!features.empty() && flag.back() != ',') {
+          features = ',' + features;
+        }
+        features = flag.str() + features;
+      }
+    } else {
+      if (flag.back() != ',') {
```

Comment:
Same thing here, we split `additional_features` by commas, so this check doesn't do naything.  We do want to check if `features.empty()` though, and only add the comma if it is non-empty.